### ISSUE TITLE
A bug in bitsback coding

### ISSUE THREAD

**Issue by bfs18** [bjlkeng/sandbox#10]
https://github.com/bjlkeng/sandbox/blob/3dac79472df6e895d6eb57f4fdbdaf2032ec0cf6/bitsback/bitsback.py#L29
scale should be standard deviation, not variance. changing scale to `scale=np.exp(0.5. * log_var)` improves the result a lot.

**Comment by bjlkeng** [comment]
Nice find, thank you!  

I'll double check once I get a few minutes and fix the notebook and blog post up.

**Comment by bfs18** [comment]
@bjlkeng Thank you for the detailed introduction and concise code. It helped me understand bitsback encoding very well

**Comment by bjlkeng** [comment]
Thanks for finding the bug @bfs18 !  I fixed the bug you mentioned and then realized I had some more issues with the encoding/decoding of the latent variables.  So I fixed those too, and now I can achieve 1.53 bits/pixel!  Much better (although not as good as the original paper). See: https://github.com/bjlkeng/sandbox/tree/master/bitsback

I also updated the blog post too for those who come across it in the future: https://bjlkeng.io/posts/lossless-compression-with-latent-variable-models-using-bits-back-coding/

Thanks again for finding that bug!
